### ISSUE TITLE
How To use the object I defined as key in hashtable

### ISSUE THREAD

**Issue by daversun** [preshing/junction#47]

**Comment by Jaimies** [comment]
I don't think this issue includes enough detail to be answered. However, I did create a new one that includes all the needed information: #48. I suggest this issue can be closed.
